docs/browser/README.md

```diff
@@ -1,3 +1,5 @@
+# @0x/mesh-browser
```

Comment:
@fabioberger this doesn't look right. There are two `@0x/mesh-browser` headings. Is this maybe a bug in the `cut-release` script?

Reply:
This looks like a faulty merge. The `cut-release` doesn't do any more extensive regexes, just version regexes.